<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerScalarVolumeDisplayWidget</class>
 <widget class="qSlicerWidget" name="qSlicerScalarVolumeDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scalar Volume Display</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="InterpolateLabel">
     <property name="text">
      <string>Interpolate:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="LookupTableLabel">
     <property name="text">
      <string>Lookup Table:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLColorTableComboBox" name="ColorTableComboBox">
     <property name="showChildNodeTypes">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="InterpolateCheckbox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="qMRMLWindowLevelWidget" name="MRMLWindowLevelWidget"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="qMRMLVolumeThresholdWidget" name="MRMLVolumeThresholdWidget"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="PresetsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLVolumeThresholdWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLVolumeThresholdWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWindowLevelWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWindowLevelWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerVolumesModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerScalarVolumeDisplayWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ColorTableComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
